src/App.tsx

```diff
@@ -1,54 +1,79 @@
-import React from 'react';
+import React, {
+  useCallback,
+  useMemo,
+  useState,
+} from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
for fix such linter issue on github, commit and push package.json file
![Screenshot from 2023-09-05 16-37-52](https://github.com/mate-academy/react_autocomplete/assets/33029864/5dc52c27-3d9b-471d-bd47-09c70ec97927)

Reply:
still not fixed, you need to write `git add package.json`
<img width="587" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/70220667/57400b04-0bf7-47c7-ab6e-663ceaf3b0b0">